### ISSUE TITLE
page is not responsive al all

### ISSUE THREAD

**Issue by devagya-rattan** [SUGAM-ARORA/UniCollab#194]
**Describe the bug**
The website's responsive design is failing on multiple device sizes. Specifically, on small screens (e.g., smartphones with a width of 320px or less), the navigation menu overlaps with the header content, 

**To Reproduce**
Navigation Menu: Implement a media query targeting screens with a width of 320px or less to adjust the styling of the navigation menu. Consider using a collapsible menu or hamburger icon to save space.
Images
: Ensure images use responsive design techniques, such as setting the max-width to 100% and height to auto, to maintain their aspect ratio and fit within the screen dimensions.
Horizontal Scrolling: 
Review the CSS for any fixed-width elements that may be causing horizontal scrolling and adjust them to be responsive. Use relative units like percentages or viewport widths (vw) instead of fixed pixels.

**Screenshots**

![Screenshot (144)](https://github.com/SUGAM-ARORA/UniCollab/assets/112056749/9393782a-f691-4af1-8480-821fe29b3470)

**Comment by devagya-rattan** [comment]
Please assign me this issue

**Comment by SUGAM-ARORA** [comment]
@devagya-rattan Issue assigned to you

**Comment by Ad2iti2003** [comment]
Please assign me this issue

**Comment by devagya-rattan** [comment]
You can work on it
I have some other work to do

**Comment by SUGAM-ARORA** [comment]
@Ad2iti2003 Are you up for this?

**Comment by Tejsvapandey1** [comment]
@SUGAM-ARORA  can i work on this

**Comment by SUGAM-ARORA** [comment]
@Tejsvapandey1 Yes you can work on this. When the required changes has been made you can tag me
